src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <!-- <link
+      rel="stylesheet"
+      href="styles/style.css"
+    /> -->
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="image"
+        src="images/imac.jpeg"
+        alt="imac"
+      />
+      <div class="content-wrapper">
+        <div class="product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="product-code">Product code: 195434</div>
+        <div class="content-revie">
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="revie-text">Reviews: 5</div>
+        </div>
+        <div class="content-price">
+          <div class="price-text">Price:</div>
+          <div class="prise-num">$2,199</div>
```

Comment:
There is a typo in the class name 'prise-num'. It should likely be 'price-num' to maintain consistency and avoid potential styling issues.